src/config/dynamic-theme-fixes.config

```diff
@@ -5007,6 +5007,33 @@ h4, h5, h6 {
 
 ================================
 
+taobao.com
+
+INVERT
+#logo > a > span
+#J_SearchIcon
+.J_ItemAmount
+.J_MyShopCoupon
+.J_BtnEditSKU
+.cart-checkbox
+
+CSS
+#logo > a > span {
+    background-color: rgba(0,0,0,0) !important;
```

Comment:
`rgba(0,0,0,0)` is `transparent` rename the value to that to make it more clear.